scripts/parser-tests/typescript/allowlist.txt

```diff
@@ -420,3 +420,4 @@ withStatement.ts
 withStatementErrors.ts
 withStatementInternalComments.ts
 withStatementNestedScope.ts
+parserConstructorDeclaration12.ts
```

Comment:
https://github.com/microsoft/TypeScript/blob/master/tests/cases/compiler/parserConstructorDeclaration12.ts
```ts
class C {
  constructor<>() { }
  constructor<> () { }
  constructor <>() { }
  constructor <> () { }
  constructor< >() { }
  constructor< > () { }
  constructor < >() { }
  constructor < > () { }
}
````

Reply:
Can you update this line: https://github.com/babel/babel/blob/bbe0cf09fc60c130fc63a7176e563781582612b8/Makefile#L3
to update the TS tests?
And then run `make test-typescript-update-allowlist`

Reply:
I've fixed at c7fa9bd! But I'm not familiar with how babel-parser tests work, so I'm not sure if it's correct.

Reply:
It's correct! Btw, `allowlist` contain tests that we don't pass and not tests that we are correctly handling.